### PR TITLE
Fix drawInFront overlays only rendering in left eye

### PR DESCRIPTION
**Pull request by SamGondelman**
Fixes a bug that caused some overlays (specifically, all line 3d overlays and dashed line cube overlays) to only render in your left eye when they had drawInFront enabled.  Also some small inconsequential fixes to the line 3d code.

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/2915eef3a31f8832e923714543f9b3eb/raw/1016fe46c76526da43b72e10d8529caad4cd1ad4/DrawInFrontTest.js).  It will create a bunch of shapes and lines down the z axis from you.  Move or create an entity so that the overlays are being blocked.
- In HMD mode, on master, you would see this (most of the overlays were not rendered in the right eye):
drawinfrontbefore
- In this build, you should see this:
drawinfrontafter

### CODE REVIEW

File: libraries/render-utils/src/RenderDeferredTask.cpp [hunk 1]
```diff
@@ -357,7 +357,7 @@ void DrawOverlay3D::run(const RenderContextPointer& renderContext, const Inputs&
         if (_opaquePass) {
             gpu::doInBatch(args->_context, [&](gpu::Batch& batch){
                 batch.enableStereo(false);
-                batch.clearFramebuffer(gpu::Framebuffer::BUFFER_DEPTH, glm::vec4(), 1.f, 0, true);
+                batch.clearFramebuffer(gpu::Framebuffer::BUFFER_DEPTH, glm::vec4(), 1.f, 0, false);
```
Comment (SamGondelman) [inline libraries/render-utils/src/RenderDeferredTask.cpp]: This is the actual fix.  We want to clear both eyes but we were only clearing whatever the viewport was set as before.  Other changes shouldn't have any real effect, but seemed beneficial.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2487/HighFidelity-Beta-PR11189-816343028203ef50eeae2bb30d26cffdd09af776.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2487/HighFidelity-Beta-PR11189-816343028203ef50eeae2bb30d26cffdd09af776.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2490/HighFidelity-Beta-PR11189-194b7d6655339ef02f6122fd769df78d9a9ae22e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2490/HighFidelity-Beta-PR11189-194b7d6655339ef02f6122fd769df78d9a9ae22e.exe

**Comment by cain-kilgore** [comment]
Testing.

**Comment by cain-kilgore** [comment]
# Test Passed
Test | Result
--- | ---
Run this. It will create a bunch of shapes and lines down the z axis from you. Move or create an entity so that the overlays are being blocked. | Pass

I was able to see the overlays in both eyes on my HMD. All good!

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Pull request merged**